neo/Prompt/Commands/Config.py

```diff
@@ -0,0 +1,205 @@
+from prompt_toolkit import prompt
+from neo.logging import log_manager
+from neo.Prompt.CommandBase import CommandBase, CommandDesc, ParameterDesc
+from neo.Prompt.Utils import get_arg
+from neo.Settings import settings
+from neo.Network.NodeLeader import NodeLeader
+import logging
+
+
+class CommandConfig(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+        self.register_sub_command(CommandConfigOutput())
+        self.register_sub_command(CommandConfigSCEvents())
+        self.register_sub_command(CommandConfigDebugNotify())
+        self.register_sub_command(CommandConfigVMLog())
+        self.register_sub_command(CommandConfigNodeRequests())
+
+    def command_desc(self):
+        return CommandDesc('config', 'configure internal settings')
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+
+        if not item:
+            print("run `%s help` to see supported queries" % self.command_desc().command)
+            return
+
+        try:
+            return self.execute_sub_command(item, arguments[1:])
+        except KeyError:
+            print(f"{item} is an invalid parameter")
+            return
+
+
+class CommandConfigOutput(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments=None):
+        return start_output_config()
+
+    def command_desc(self):
+        return CommandDesc('output', 'configure the output level settings')
+
+
+class CommandConfigSCEvents(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        c1 = get_arg(arguments)
+        if c1 is not None:
+            c1 = c1.lower()
+            if c1 == 'on' or c1 == '1':
+                settings.set_log_smart_contract_events(True)
+                print("Smart contract event logging is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_log_smart_contract_events(False)
+                print("Smart contract event logging is now disabled")
+                return c1
+            else:
+                print("Cannot configure log. Please specify on|off")
+                return
+        else:
+            print("Cannot configure log. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('sc-events', 'toggle smart contract events on or off', [p1])
```

Comment:
```suggestion
         return CommandDesc('sc-events', 'toggle printing smart contract events', [p1])
```
the possible options are given by the `ParameterDesc`